nbs/methods.ipynb

```diff
@@ -44,6 +44,7 @@
     "\n",
```

Comment:
<p>The <code>nbs/methods.ipynb</code> file is intended to work only with numpy arrays. If you want to test a large summing matrix, please create it from scratch using numpy instead of downloading and importing a dataframe. Since the inclusion of <code>CodeTimer</code> is not reviewed yet, you could test the performance of the change using the <code>time</code> module:</p><p> </p><pre class="ql-syntax" spellcheck="false">from time import time

# performace wo sparsity
init_wo_sparsity = time()
cls_bu = BottomUp()
cls_bu.reconcile(...)
end_wo_sparsity = time()

# performance sparsity
init_sparsity = time()
cls_bu = BottomUp(sparsity=True)
cls_bu.reconcile(...)
end_sparsity = time()

# test sparsity helps
assert (end_sparsity - init_sparsity) &lt; (end_wo_sparsity - init_wo_sparsity)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/hierarchicalforecast/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>